scripts/create-symlinks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage in the project root folder:
+# ./scripts/create-symlinks.sh PACKAGE_NAME
+#
+# Usage in package folder
+# ../../scripts/create-symlinks.sh
+
+packageDir=".";
+if [ -n "$1" ]
+	then
+		# Override $pageDir if argument was provided.
+		packageDir="./packages/$1";
+
+		# Just in case package folder doesn't exist yet.
+		mkdir -p $packageDir;
+fi
+
+declare -a arr=(".prettierrc.json" ".prettierignore" ".npmrc" ".nycrc" ".npmignore" ".babelrc" ".eslintignore" "cypress.json" "cypress" "browsertest")
+
+for i in "${arr[@]}"
+do
+   ln -s ../../templates/$i.tmpl $packageDir/$i;
+   printf "Symlink Created: $packageDir/$i -> ../../templates/$i.tmpl\n";
```

Comment:
Not sure we need feedback in the success case here.

Reply:
I think it's better to have this in case of a typo and would be good to see if they were created during CI process ?